target/linux/ramips/mt7621/base-files/etc/board.d/01_leds

```diff
@@ -23,6 +23,11 @@ d-team,newifi-d2)
 	ucidef_set_led_netdev "wlan2g" "WiFi 2.4GHz" "blue:wlan2g" "wlan0"
 	ucidef_set_led_netdev "wlan5g" "WiFi 5GHz" "blue:wlan5g" "wlan1"
 	;;
+dlink,dir-853-r1)
+	ucidef_set_led_netdev "internet" "internet" "blue:net" "wan"
+	ucidef_set_led_netdev "wlan2g" "WiFi 2.4GHz" "blue:wlan2g" "wlan0"
+	ucidef_set_led_netdev "wlan5g" "WiFi 5GHz" "blue:wlan5g" "wlan1"
```

Comment:
These two lines should be replaced by a trigger in the DTS file.

Reply:
Triggers do not work with DBDC and only one wireless card LED works if it is using triggers.

Reply:
I doubt that you are the only one here where triggers don't work. Which did you try?

Reply:
> I doubt that you are the only one here where triggers don't work. Which did you try?

it didn't work for me.
here's what i did:

target / linux / ramips / mt7621 / base-files / etc / board.d / 01_leds

;;
dlink, dir-853-r1)
ucidef_set_led_netdev "internet" "internet" "blue: net" "wan"
;;

target / linux / ramips / dts / mt7621_dlink_dir-853-r1.dts

wlan2g {
label = "blue: wlan2g";
gpios = <& gpio 4 GPIO_ACTIVE_LOW>;
linux, default-trigger = "phy0tpt";
};

wlan5g {
label = "blue: wlan5g";
gpios = <& gpio 3 GPIO_ACTIVE_LOW>;
linux, default-trigger = "phy1tpt";
};

Reply:
Try phy0radio and phy1radio ...

Reply:
Okay I will try.

Reply:
> phy0radio and phy1radio

phy0radio and phy1radio worked for me thanks.